jt-jiffle/jt-jiffle-op/src/main/java/it/geosolutions/jaiext/jiffleop/JiffleDescriptor.java

```diff
@@ -150,25 +177,32 @@ public int getNumSources() {
 
     /**
      * RenderedOp creation method that takes all the parameters, passes them to the
-     * ParameterBlockJAI and then call the JAI create method for the mosaic
-     * operation with no data support.
+     * ParameterBlockJAI and then call the JAI create method for the mosaic operation with no data
+     * support.
      *
      * @param sources The RenderdImage source array used for the operation.
-     * @param sourceImageNames The array of source image names, that will be referred from the script. Can be null, in such case "src, src1, src2, ..." will be used as image names
-     * @param destName The name of the destination image. Can be null, in such case "dest" will be used
-     * @param destBounds The output bounds. It is required only if there are no sources, and no {@link javax.media.jai.ImageLayout} is provided in the hints, otherwise can be null.
-     * @param destType The destination type. Not required, will default to {@link DataBuffer#TYPE_DOUBLE}
-     * @param sourceCoordinateTransforms The world to image coordinate transforms for the sources                
+     * @param sourceImageNames The array of source image names, that will be referred from the
+     *     script. Can be null, in such case "src, src1, src2, ..." will be used as image names
+     * @param destName The name of the destination image. Can be null, in such case "dest" will be
+     *     used
+     * @param destBounds The output bounds. It is required only if there are no sources, and no
+     *     {@link javax.media.jai.ImageLayout} is provided in the hints, otherwise can be null.
+     * @param destType The destination type. Not required, will default to {@link
+     *     DataBuffer#TYPE_DOUBLE}
+     * @param destBands The number of destination bands. Not required, will be automatically computed from the script source unless the  
```

Comment:
unless the.... ?? :)

Reply:
Suspense! Fixing it :-D
It's " the destination bands specs are using  expressions", e.g. it could be something like 
``dest[x() % 2 == 0? 1 : 2]``